cypress/e2e/a11y.cy.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="Cypress" />
+import '../support/e2e.js'
+
+describe('A11y Test Suite', () => {
+    
+    beforeEach(() => {
+        cy.visit('/')
+        cy.injectAxe()
+    })
+    
+    it('finds no a11y issues in light mode, considering customer details and address', () => {
+    
+            cy.checkA11y()
+        cy.get('tbody tr').first().click()
+        cy.get('.show-address-btn').click()
+            cy.checkA11y()
+    })
+    
+    it('finds no a11y issues in dark mode, considering customer details and address', () => {
+    
+        cy.get('#theme-toggle-button').click()
+            cy.checkA11y()
+        cy.get('tbody tr').first().click()
+        cy.get('.show-address-btn').click()
+            cy.checkA11y()
+    })
+
+})
```

Comment:
```suggestion
})

```

É uma convenção deixar uma linha em branco ao final de cada arquivo. Isso evita que ferramentas como o GitHub exibam o ícone "No newline at end of file".